playwright-e2e/pages/add-application-documents.ts

```diff
@@ -0,0 +1,44 @@
+import { type Page, type Locator, expect } from "@playwright/test";
+import config from "../settings/test-docs/config";
+import path from "path";
+
+export class AddApplicationDocuments {
+
+    readonly page: Page;
+    readonly applicationDocumentsHeading: Locator;
+    readonly addNewButton: Locator;
+    readonly typeOfDocument: Locator;
+    readonly chooseFileButton: Locator;
+    readonly giveDetailsText: Locator;
+    readonly continueButton: Locator;
+    readonly checkYourAnswersHeader: Locator;
+    readonly saveAndContinueButton: Locator;
```

Comment:
checkYourAnswersHeader, saveAndContinueButton, continueButton can be defined in a generic class to be reused rather than defined in each individual test.

Reply:
updated